src/screens/WebView/index.tsx

```diff
@@ -63,9 +57,10 @@ export default function WebView({ isLogin }: Props) {
     if (webview) {
       const loadstop = () => {
         setLoading({ ...loading, refresh: false })
-
+        // Ignore the login handling if not on login page
+        if (pathname !== '/') return
```

Comment:
Even though I know this works fine, I think it's not good to read without the curly braces.
Took me a few seconds to understand why we had the else statement since I could not see where the first if finished.

Reply:
alright!
If we want to stick with that, would there be any way to have that as a rule in eslint? It's a habit of mine and I'm sure others do it too.